managed/Makefile

```diff
@@ -22,6 +22,13 @@ PMM_LD_FLAGS = -ldflags " \
 			-X 'github.com/percona/pmm/version.Branch=$(PMM_RELEASE_BRANCH)' \
 			"
 
+IS_ARM        := $(filter arm64,$(shell uname -m))
```

Comment:
Maybe we should include `aarch64` in the filter list too?

Reply:
Good idea!

Do you know what type of hardware we could get it from? I've checked my M2 processor and only got `arm64`.

Reply:
I think it must be RaspberryPi, right? In such case I doubt it's a good idea to run PMM Server on a Pi machine, it'd not have sufficient resources to be performant.